.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
We haven't been adding this (in general) to packages.  Add it to ~/.npmrc if needed.

Reply:
Right, snuck in. will remove.

Reply:
Removed.